docs/Manpage.md

```diff
@@ -1936,7 +1936,7 @@ example, run `export HOMEBREW_NO_INSECURE_REDIRECT=1` rather than just
   *Default:* `native`.
 
 - `HOMEBREW_ARTIFACT_DOMAIN`
-  <br>Prefix all download URLs, including those for bottles, with this value. For example, `HOMEBREW_ARTIFACT_DOMAIN=http://localhost:8080` will cause a formula with the URL `https://example.com/foo.tar.gz` to instead download from `http://localhost:8080/example.com/foo.tar.gz`.
+  <br>Prefix all download URLs, including those for bottles, with this value. For example, `HOMEBREW_ARTIFACT_DOMAIN=http://localhost:8080` will cause a formula with the URL `https://example.com/foo.tar.gz` to instead download from `http://localhost:8080/https://example.com/foo.tar.gz`. Bottle URLs however, have their domain replaced with this prefix. Using the same value for example, would cause data hosted under `https://ghcr.io/v2/homebrew/core/someformula/blobs/sha256:0110100001101111011011010110010101100010011100100110010101110111` to be instead downloaded from `http://localhost:8080/v2/homebrew/core/someformula/blobs/sha256:0110100001101111011011010110010101100010011100100110010101110111`
```

Comment:
Thanks for the PR! Unfortunately, this change needs made at https://github.com/Homebrew/brew/blob/master/Library/Homebrew/env_config.rb#L22-L26 instead.